.build/docker/run-tests.sh

```diff
@@ -69,8 +69,11 @@ if [[ ! "${java_version}" =~ $regx_java_version ]]; then
     exit 1
 fi
 
-python_version="3.6"
-command -v python >/dev/null 2>&1 && python_version="$(python -V | awk '{print $2}' | awk -F'.' '{print $1"."$2}')"
+# allow python version override, otherwise default to current python version or 3.7
+if [ "x" == "x${python_version}" ] ; then
+    command -v python >/dev/null 2>&1 && python_version="$(python -V 2>&1 | awk '{print $2}' | awk -F'.' '{print $1"."$2}')"
+    python_version="${python_version:-3.7}"
```

Comment:
I don't have a horse in this race, but why 3.7? vs. 3.8, 3.11, etc.

Reply:
just a conservative/minimal bump after dropping 3.6

(am all for discussing removing 3.7 too (in a separate ticket))